physionet-django/project/views.py

```diff
@@ -620,14 +622,54 @@ def project_content(request, project_slug, **kwargs):
         instance=project)
     reference_formset = ReferenceFormSet(instance=project)
 
+    # Creates forms for each section of this project's content type
+    section_forms = []
+    sections = ProjectSection.objects.filter(resource_type=project.resource_type).order_by('default_order')
+    for s in sections:
+        try:
+            # Try to get currently existing content for section
+            section_content = project.project_content.get(project_section=s)
+            section_forms.append(forms.SectionContentForm(instance=section_content))
+        except:
+            # Creates form with empty instace in case content is not found
```

Comment:
Please could you add the exception type? 

> PEP8: "When catching exceptions, mention specific exceptions whenever possible instead of using a bare except". https://www.python.org/dev/peps/pep-0008/#id51.